sdks/ios/ZelloChannelDemo/ZelloChannelDemo/Views/RiderTalkViewController.m

```diff
@@ -73,4 +114,17 @@ - (void)session:(ZCCSession *)session incomingVoiceDidStop:(ZCCIncomingVoiceStre
   [self setButtonNormal];
 }
 
+- (void)session:(ZCCSession *)session didReceiveLocation:(ZCCLocationInfo *)location from:(NSString *)sender {
+  [self showDriver:sender location:location];
+}
+
+- (void)showDriver:(NSString *)driver location:(ZCCLocationInfo *)location {
```

Comment:
I do not see any other callers of this method. This separation may be logical but seems premature, just a thought not required by maybe skinny up the code a bit by deleting lines 118-121.

Reply:
Done